opal/util/info.c

```diff
@@ -87,7 +87,8 @@ int opal_info_dup (opal_info_t *info, opal_info_t **newinfo)
         OBJ_RETAIN(iterator->ie_key);
         newentry->ie_value = iterator->ie_value;
         OBJ_RETAIN(iterator->ie_value);
-     }
+        opal_list_append (&((*newinfo)->super), (opal_list_item_t *) newentry);
+    }
```

Comment:
Weird whitespace change... any idea how that happened in the cherry pick?

Reply:
I think that is because master was already `clang-format`ed and I figured that is the correct position of the closing bracket. I can take out the whitespace change though